src/main/groovy/de/dkfz/roddy/execution/jobs/SubmissionCommand.groovy

```diff
@@ -65,7 +82,7 @@ abstract class SubmissionCommand extends Command {
 
         // collect parameters for job submission
         List<String> parameters = []
-        parameters << getEnvironmentExportParameter()
+        parameters << assembleVariableExportParameters()
```

Comment:
Separation is not useful. See PBS implementation, where `-V` and `-v` options are tightly related.